department-of-veterans-affairs/abd-vro
# Use SecRel 4.0

## What was the problem?


A release candidate for major version 4.0.0 has been released.
https://dsva.slack.com/archives/C03UA9MV1EH/p1675107172862209
> Teams must migrate to v4.0.0 by Feb 14th

Associated tickets or Slack threads:
- #888

## How does this fix it?[^1]

Creates new GH Action workflow to use SecRel 4. Also update libraries to address new SecRel alerts.

TODO: Once it's working consistently, disable SecRel 3 workflow.

## How to test this PR
- Manually run workflow [^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
